### PR TITLE
fix: disable log throttling if primary log level is debug

### PR DESCRIPTION
**Pull request by SergeTupchiy**
Fixes EMQX-11530

Release version: v5.6

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: rel/i18n/emqx_conf_schema.hocon [hunk 1]
```diff
@@ -480,10 +480,12 @@ desc_log_throttling.label:
...
 desc_log_throttling.desc:
 """Log throttling feature reduces the number of potentially flooding logged events by
-dropping all but the first event within a configured time window."""
+dropping all but the first event within a configured time window.
+The throttling is automatically disabled if the primary log level is set to debug."""
 
 log_throttling_time_window.desc:
-"""For throttled messages, only log 1 in each time window."""
+"""For throttled messages, only log 1 in each time window.
+If the primary log level is debug, no log messages are throttled."""
```
Comment (zmstone) [inline rel/i18n/emqx_conf_schema.hocon:488]:
```suggestion
"""For throttled messages, only log 1 in each time window."""
```

File: rel/i18n/emqx_conf_schema.hocon [hunk 2]
```diff
@@ -480,10 +480,12 @@ desc_log_throttling.label:
 
 desc_log_throttling.desc:
 """Log throttling feature reduces the number of potentially flooding logged events by
-dropping all but the first event within a configured time window."""
+dropping all but the first event within a configured time window.
+The throttling is automatically disabled if the primary log level is set to debug."""
```
Comment (zmstone) [inline rel/i18n/emqx_conf_schema.hocon:484]: there is no 'primary log level' concept exposed to users.

```suggestion
dropping all but the first event within a configured time window.
The throttling is automatically disabled if `console` or `file` log level is set to `debug`."""
```

Comment (SergeTupchiy) [reply rel/i18n/emqx_conf_schema.hocon]: Indeed, my bad.. Fixed

File: rel/i18n/emqx_conf_schema.hocon (hunk 1 above)
Comment (SergeTupchiy) [reply rel/i18n/emqx_conf_schema.hocon]: Fixed

**Review by savonarola — APPROVED** [review]

**Pull request merged**